samples/FunctionApp/FunctionApp.csproj

```diff
@@ -15,7 +15,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Functions.Worker.Extensions.ApplicationInsights" Version="1.0.0-preview3" />
-    <PackageReference Include="Microsoft.Azure.Functions.Worker.Sdk" Version="1.0.1" />
+    <PackageReference Include="Microsoft.Azure.Functions.Worker.Sdk" Version="1.0.3" OutputItemType="Analyzer" />
```

Comment:
Just curious, what is the purpose of adding - OutputItemType="Analyzer" here?

Reply:
Changes from [this PR for Debugging](https://github.com/Azure/azure-functions-dotnet-worker/pull/616) - leaving it for now because I may need to locally debug this branch still, but will remove before merging. Hopefully that other PR will be merged soon so I can rebase this branch.

Reply:
Got it. Thanks.